client/my-sites/plans/current-plan/jetpack-checklist/index.tsx

```diff
@@ -278,7 +278,7 @@ class JetpackChecklist extends PureComponent< Props & LocalizeProps > {
 						completedButtonText={ translate( 'Change', { context: 'verb' } ) }
 						completedTitle={ translate( 'You turned on Downtime Monitoring.' ) }
 						description={ translate(
-							"Monitor your site's uptime and alert you the moment downtime is detected with instant notifications."
+							'Jetpack will continuously watch your site, and alert you the moment that downtime is detected with instant notifications.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Jetpack will continuously monitor your site, and alert you the moment downtime is detected.' )` *ES Score: 6*